### ISSUE TITLE
Plugin blocks everything but the ability to create claims, *if* you can get a gold shovel that is.

### ISSUE THREAD

**Issue by PairedPrototype** [bloodmc/GriefDefender#240]
I'm trying this out before I buy, but so far it's not looking good. In text, it does everything I want/need but in practice, it just seems completely broken.

I've built the latest code from the master branch tagged as 1.4.4.
I run a SpongeForge server, the wiki doesn't give a specific minimum version of Sponge to work with, so since it says "up-to-date" I'm using `spongeforge-1.12.2-2838-7.2.3`.
For LuckPerms I've got `LuckPerms-Sponge-5.1.75`.
For Forge I have `forge-1.12.2-14.23.5.2854`.
These are the only plugins/mods I have loaded since I disabled all the others for the purpose of trying to debug these issues.

I used the server console to grant the default group `griefdefender.user.*` and the admin group `griefdefender.admin.*`.
But attempting to build in-game throws out:
> You don't have the wilderness's permission to do to interact with the item minecraft:stone.

Okay, so I attempt to ignore it by typing in chat `/gd claim ignore` but:
> The command gd has been blocked by claim owner wilderness

Fine, I'll use the console... 
> Error: Console may not execute this command.

Turns out there's no option to target a player anyway. 

I used the console to set my player into creative mode and then I was able to grab a stick and gold shovel. I can create a claim using the shovel and identify it using the stick, but while within the claim I still cannot build or execute commands, I get the same errors as above just with "wilderness" replaced with my username.

I tried checking if it's a perms issue, but running an `lp check` on my user for all `griefdefender.admin.claim.command.ignore` nodes return `true`, including the `griefdefender.admin.claim.command.ignore.wilderness` node. I've tried running `/gdreload` from the console and restarting the server but still no change 🙁

How is this plugin meant to work?
I feel like I'm missing something, but the wiki is to suggest otherwise.

**Comment by bloodmc** [comment]
Judging from the issues you are seeing, it sounds like your LuckPerms setup is completely broken. See https://github.com/bloodmc/GriefDefender/wiki/Permissions#important
Your default luckperms group must inherit from `default` or you will experience many permission issues with GD.

If you really want to test GD then I suggest you setup a clean server with only LP/GD/SF to rule out any other issues.

**Comment by PairedPrototype** [comment]
You're right, starting a fresh server and using a local LuckPerms DB store instead with everything at default seems to allow me to build and break blocks. Thank you for pointing me in the right direction!

**Comment by PairedPrototype** [comment]
After some deeper digging, I've discovered the issue. My groups are indeed inheriting from the `default` group, however, when I first setup LuckPerms around 2 years ago, I wanted really fine-grained controller over the permissions and the first permission I set was `* false`. In hindsight, this might have been a terrible way to go about it. Removing this appears to have solved the issue I was experiencing.

TLDR; Check you haven't negated `*` on your groups 👍
